src/Sylius/Component/Core/spec/Promotion/Filter/ProductFilterSpec.php

```diff
@@ -53,4 +53,9 @@ function it_returns_all_items_if_configuration_is_invalid(OrderItemInterface $it
     {
         $this->filter([$item], [])->shouldReturn([$item]);
     }
+
+    function it_returns_all_items_if_configuration_is_empty(OrderItemInterface $item)
```

Comment:
I wonder whether it is a desired behaviour. There should be no filter at all first if we want to pass through all the items.